tasks/build_tags.py

```diff
@@ -53,6 +53,7 @@
     "zlib",
     "zstd",
     "test",  # used for unit-tests
+    "goexperiment.systemcrypto",  # used for FIPS mode
```

Comment:
I'm surprised this was not needed for linux? Is this a leftover/something that was missing, or is it indeed windows specific?

Reply:
I was surprised, too. but without it I get this error from `inv agent.build --flavor fips`
```
Warning: unknown build tag 'goexperiment.systemcrypto' was filtered out from included tags list.
```

Reply:
oh, I see, the fips tags are added after it gets removed, so the Agent is still built correctly
https://github.com/DataDog/datadog-agent/blob/2d20432a10d9a0cec68a029e74f16c1bf39dd2cc/tasks/agent.py#L198-L199